.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 examples/**/node_modules
 examples/**/dist
+**/*.test.js
```

Comment:
that's a bold move

Reply:
forgot to remove this (the test linting was a mess and I realised I was pretty much wasting time on it)